### PR TITLE
:fire: Remove redis, websockets, sse, and webhooks services

### LINKED ISSUE #962: ⚡ Remove Webhooks services
The Webhooks service currently contains the following services that run concurrently:
- AcapyEventsProcessor
- SseManager
- BillingManager

These are each important enough to exist as their own service.

Edit: after #1095 , these services can be removed!

### PR DESCRIPTION
**Pull request by ff137**
:fire: :fire: :fire:

Removes redis-related scripts, config, and services.
Removes websocket endpoint, service and tests. And websocket related dependencies.
Removes deprecated webhooks endpoint.
Removes the sse functions that still used Webhooks URL. (Keeps the one that uses Waypoint)
And importantly, deletes all of the webhook module, and governance-webhooks-web dependency

:fire: :fire: :fire: 

Related charts PR: https://github.com/didx-xyz/charts/pull/301

Closes #962

Side notes: 
- uvicorn no longer installs with `standard` extra packages (no need for it -- it included a websocket dependency we no longer use).
- ~~there's a deleted Webhooks.md doc that should be replaced with updated documentation~~ (re-added)
- deleted env vars: 
  - Webhooks related: `WEBHOOKS_TOPIC_ALL`, `WEBHOOKS_URL`, `WEBHOOKS_PUBSUB_URL`
  - SSE related: `MAX_EVENT_AGE_SECONDS`, `MAX_QUEUE_SIZE`, `QUEUE_CLEANUP_PERIOD`, `CLIENT_QUEUE_POLL_PERIOD`, `SET_LOCKS`
  - LAGO related: `LAGO_URL`, `LAGO_API_KEY` 
  - Redis related: `REDIS_NODES`
- Updated openapi docs (and the main docs, so they no longer reference webhooks pod)
- Removed `look_back` SSE parameter. Can maybe be added later if there's need for it.

### CODE REVIEW

**Comment by ff137** [comment]
For review: please search for webhooks, sse, redis, and websocket references in codebase, and pls confirm relevant stuff is removed 🙏

File: .github/workflows/continuous-deploy.yml [hunk 1]
```diff
@@ -1163,6 +1158,7 @@ jobs:
             -e CLIENT_SECRET=${{ secrets.CLIENT_SECRET }} \
             -e GOVERNANCE_CLIENT_SECRET=${{ secrets.GOVERNANCE_CLIENT_SECRET }} \
             -e CLOUDAPI_URL=${{ secrets.CLOUDAPI_URL }} \
+            -e WAYPOINT_URL=${{ secrets.WAYPOINT_URL }} \
```
Comment (ff137) [inline .github/workflows/continuous-deploy.yml:1161]: @rblaine95 I added reference to waypoint_url secret here, which doesn't yet exist - please can you add 🙏

File: scripts/k6/libs/functions.js [hunk 2]
```diff
@@ -755,7 +755,7 @@ export function genericWaitForSSEEvent(config) {
     sseTag
   } = config;
 
-  const sseUrl = `${__ENV.CLOUDAPI_URL}/tenant/v1/sse/${walletId}/${sseUrlPath}/${threadId}/${eventType}`;
+  const sseUrl = `${__ENV.WAYPOINT_URL}/sse/${walletId}/${sseUrlPath}/${threadId}/${eventType}`;
```
Comment (ff137) [inline scripts/k6/libs/functions.js:758]: test whether waypoint url works here

**Review by ff137 — COMMENTED** [review]
Still to do:
- confirm relevant references removed
- update openapi docs
- @cl0ete please take a look at the deleted docs/Webhooks.md. We can perhaps take that old doc and refactor into Waypoints doc

File: .github/workflows/continuous-deploy.yml (hunk 1 above)
Comment (rblaine95) [reply .github/workflows/continuous-deploy.yml:1161]: Done 👍

**Review by cl0ete — APPROVED** [review]
Nice! I still see some reference to webhooks and sse in openapi yaml/json files.

**Pull request merged**